src/RiakClient/Commands/CommandOptions.cs

```diff
@@ -25,122 +7,30 @@ namespace RiakClient.Commands
     /// </summary>
     public abstract class CommandOptions : IEquatable<CommandOptions>
     {
-        private readonly RiakString bucketType;
```

Comment:
Look at all the code you saved!  
![image](https://cloud.githubusercontent.com/assets/1438/13859249/f006b126-ec58-11e5-9324-2fa614868255.png)